content/5_Plat/Bitsets.mdx

```diff
@@ -74,6 +77,57 @@ void init() {
 }
 ```
 
+</CPPSection>
+<PySection>
+
+```py
+class DisjointSet:
```

Comment:
put this class in a codesnip

Reply:
why? the C++ version (above) isn't in a codesnip

Reply:
Done. I put DSU into a codesnip

Reply:
> why? the C++ version (above) isn't in a codesnip

that one should be in a snip as well then